### PR TITLE
Removes last UMDs from the modules.

### PR DESCRIPTION
**Pull request by yurydelendik**

### CODE REVIEW

File: src/main_loader.js [hunk 1]
```diff
@@ -13,60 +13,20 @@
...
-    factory((root.pdfjsMainLoader = {}), root.pdfjsDisplayAPI,
-      root.pdfjsDisplayAnnotationLayer, root.pdfjsDisplayTextLayer,
-      root.pdfjsDisplayDOMUtils, root.pdfjsSharedUtil, root.pdfjsDisplaySVG,
-      root.pdfjsDisplayGlobal);
-  }
-}(this, function (exports, displayAPI, displayAnnotationLayer,
-                  displayTextLayer, displayDOMUtils, sharedUtil,
-                  displaySVG, displayGlobal) {
-
-  // Sync the exports below with ./pdf.js file/template.
```
Comment (Snuffleupagus) [inline src/main_loader.js]: Unless I'm missing something, shouldn't we should keep this comment?

**Review by Snuffleupagus — APPROVED** [review]
Besides the inline question, is there any reason for not updating https://github.com/mozilla/pdf.js/blob/master/src/pdf.js to the same format as is now used in `src/main_loader.js`?

This looks good to me, thank you!

File: src/main_loader.js (hunk 1 above)
Comment (yurydelendik) [inline src/main_loader.js]: We can remove main_loader.js

**Pull request merged**